agent/config/default.go

```diff
@@ -26,6 +26,10 @@ func DefaultRPCProtocol() (int, error) {
 // todo(fs): IMO, this should be the definitive default for all configurable values
 // todo(fs): and whatever is in here should clobber every default value. Hence, no sourcing.
 func DefaultSource() Source {
+	cfg := consul.DefaultConfig()
+	serfLAN := cfg.SerfLANConfig.MemberlistConfig
+	serfWAN := cfg.SerfWANConfig.MemberlistConfig
```

Comment:
What's the significance of these moving from `DefaultConsulSource` to here? This seems like a better place but I don't recall offhand the details of how these two methods interact...

Reply:
In depth explanation in my other comment.

DefaultConsulSource = non user modifiable
DefaultSource = user modifiable